src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3RequestStream.cs

```diff
@@ -227,7 +203,7 @@ public async Task<HttpResponseMessage> SendAsync(CancellationToken cancellationT
                 if (useEmptyResponseContent)
                 {
                     // Drain the response frames to read any trailing headers.
-                    await DrainContentLength0Frames(requestCancellationSource.Token).ConfigureAwait(false);
+                    await DrainContentLength0Frames(cancellationToken).ConfigureAwait(false);
```

Comment:
Should this be `_requestBodyCancellationSource`?

Reply:
aahh... I missed this one, good catch!